### DISCUSSION TITLE
v4.0.2

### DISCUSSION THREAD

**Discussion by RussKie** [gitextensions/gitextensions#10646] [General]
Maintenance v4 release
![Downloads](https://img.shields.io/github/downloads/gitextensions/gitextensions/v4.0.2/total.svg)

https://github.com/gitextensions/gitextensions/blob/release/4.0/GitUI/Resources/ChangeLog.md#version-402-15-jan-2023

Requires: **.NET 6.0 Desktop Runtime** [v6.0.13](https://dotnet.microsoft.com/download/dotnet/thank-you/runtime-desktop-6.0.13-windows-x64-installer) or later

<hr /><em>This discussion was created from the release <a href='https://github.com/gitextensions/gitextensions/releases/tag/v4.0.2'>v4.0.2</a>.</em>

**Comment by c2ssb2** [comment]
Thank you so much for making the best Git client ever made (even comparing to commercial products) where I can't find any product with same rich-features (not even close).
But I have one question: Is it necessary to use new .Net framework, even with minor releases? I can't recall any software requires to install new .net framework with its minor releases, even Microsoft products.
Thanks again for a wonderful piece of art, Git Extensions!

**Reply by RussKie** [reply]
> Thank you so much for making the best Git client ever made (even comparing to commercial products) where I can't find any product with same rich-features (not even close).

You're welcome :)



> Is it necessary to use new .Net framework, even with minor releases?

If you look at the [.NET release history](https://dotnet.microsoft.com/download/dotnet/6.0) you'll see that pretty much every servicing release is a "security release" (meaning it addresses some security vulnerability). Not seldom, those also carry fixes for the .NET Desktop runtime.
So, we target the latest .NET runtime to ensure we take the fixes and run on the most secure version of the runtime available to us at the time of a build. This also should help our users to stay current as well.

**Reply by c2ssb2** [reply]
Thanks for your kind reply, and for best Git client ever existed!
